app/controllers/api/accounts_controller.rb

```diff
@@ -37,6 +37,7 @@ def create
   # PATCH /api/accounts/:id?Query
   def update
     authorize account
+    account_move_handler.move(params_group) if team_changing?
```

Comment:
how about account.group.team_id_changed?
I think this should be available already

Reply:
But the team_id of the group didn't change. We moved the account, not the group.
`account.group.team_id_changed? => false`
`account.group_id_changed? => true`
But just because the group_id changed doesn't mean that the account is in a new team

Reply:
of course ... didn't think it through completely.

Reply:
I think it's pretty ok if we just check if the group has changed. since we're taking care of an eventual team change inside AccountMoveHandler.
if account.group_id_changed? or even account.group_changed?